post_installation_configuration/node-tasks.adoc

```diff
@@ -62,7 +62,7 @@ include::modules/machine-node-custom-partition.adoc[leveloffset=+2]
 
 Understand and deploy machine health checks.
```

Comment:
Out of scope for this PR, but I don't think this brief intro is consistent with the other intros in style.

Reply:
Yeah that's on the reorg to-do list as well :smile: